api-tests/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Saleor API tests",
+  "scripts": {
+    "test": "vitest",
+    "lint": "eslint 'src/**/*.@(tsx|ts|jsx|js)' --fix",
+    "check-types": "tsc --noEmit",
+    "generate": "graphql-codegen ",
+    "fetch-schema": "export $(cat .env.local| xargs) && curl $GRAPHQL_RAW > graphql/schema.graphql"
```

Comment:
That's a very dangerous way of getting `GRAPHQL_RAW`, you are already using a javascript eco-system here, simply write a safe `.js`  script to retrieve the schema